### ISSUE TITLE
Couldn't connect to accessibility bus: Failed to connect to socket: Connection refused

### ISSUE THREAD

**Issue by kuriel-trivu** [Ulauncher/Ulauncher#580]
### Steps to reproduce
1. Install LXD on Ubuntu
2. Run a LXD Container (Ubuntu 18.04 for example)
3. Install ULauncher inside LXD Container instance

### Current behavior
Try to run ULauncher but it fails.

### Expected behavior
ULauncher works

### Environment
**Ulauncher version**: 5.8.0_all
**Linux distro**: Ubuntu 20.04 LXD Ubuntu 18.04
**Desktop environment**: none

### Logs

https://gist.github.com/kuriel-trivu/aa3038dc19b374303e1e07308ee48186

**Comment by kuriel-trivu** [comment]
ULauncher it's very useful to run into LXD since LXD most of times does not have Desktop Environment, so ULauncher can epic easy to launch an app.

**Comment by kuriel-trivu** [comment]
If there is no way to run in GUI with LXD, maybe this can be a feature request: 

Run and autocomplete apps to run from terminal, kind of https://github.com/pindexis/marker but marker it's a snippet for terminal, does not recognize the available applications.

**Comment by troycurtisjr** [comment]
Aren't LXD containers specifically designed to accomplish this kind of isolation? I think the issue here is that ulauncher is prevented by the LXD from access the dbus daemon and it will have to be the LXD container itself that either provides an alternate dbus daemon or a proxy into the one running in the session.
